api/src/main/java/com/epam/pipeline/manager/datastorage/providers/gcp/GSBucketStorageHelper.java

```diff
@@ -471,11 +473,14 @@ private void deleteAllVersions(final String bucketName, final String path, final
         final Page<Blob> blobs = client.list(bucketName,
                 Storage.BlobListOption.versions(true),
                 Storage.BlobListOption.prefix(path));
-        blobs.iterateAll().forEach(blob -> deleteBlob(bucketName, path, blob));
+        blobs.iterateAll().forEach(blob -> deleteBlob(blob, gcpClient.buildStorageClient(region), true));
```

Comment:
Can't we reuse client from methods argument?

Reply:
fixed